### DISCUSSION TITLE
Streampipes Backend API CORS policies for custom dashboards

### DISCUSSION THREAD

**Discussion by mihau1211** [apache/streampipes#2433] [Q&A] [answered]
Hi all,

I'm creating a custom dashboard using an external React-based app, mostly for using data from Datalakes on custom dashboards. The issue I'm facing is that I can't request backend API because of CORS policy. Even if I'm using mode: 'no-cors', I'm receiving 401 (I'm using headers to auth like in the related #2424).

Is there any possibility of changing the CORS policy in the Streampipes on my own? Or, maybe, have you any workaround for that issue? Will be grateful for any help.

**Comment by dominikriemer** [comment, marked as answer]
Hi @mihau1211 in case you're trying to reach the backend API from the React UI directly, you can configure a proxy in the web server to avoid the CORS problem. In this case, the problem should disappear.
